docs/intro/tutorial.rst

```diff
@@ -55,318 +55,333 @@ This will create a ``tutorial`` directory with the following contents::
+    2016-09-20 14:48:00 [scrapy] DEBUG: Telnet console listening on 127.0.0.1:6023
+    2016-09-20 14:48:00 [scrapy] DEBUG: Crawled (404) <GET http://quotes.toscrape.com/robots.txt> (referer: None)
+    2016-09-20 14:48:00 [scrapy] DEBUG: Crawled (200) <GET http://quotes.toscrape.com/page/1/> (referer: None)
+    2016-09-20 14:48:01 [quotes] DEBUG: Saved file quotes-1.html
+    2016-09-20 14:48:01 [scrapy] DEBUG: Crawled (200) <GET http://quotes.toscrape.com/page/2/> (referer: None)
+    2016-09-20 14:48:01 [quotes] DEBUG: Saved file quotes-2.html
+    2016-09-20 14:48:01 [scrapy] INFO: Closing spider (finished)
+    ...
 
-    2016-09-01 16:51:27 [scrapy] INFO: Scrapy started (bot: tutorial)
-    2016-09-01 16:51:27 [scrapy] INFO: Overridden settings: {...}
-    2016-09-01 16:51:27 [scrapy] INFO: Enabled extensions: ...
-    2016-09-01 16:51:27 [scrapy] INFO: Enabled downloader middlewares: ...
-    2016-09-01 16:51:27 [scrapy] INFO: Enabled spider middlewares: ...
-    2016-09-01 16:51:27 [scrapy] INFO: Enabled item pipelines: ...
-    2016-09-01 16:51:27 [scrapy] INFO: Spider opened
-    2016-09-01 16:51:27 [scrapy] INFO: Crawled 0 pages (at 0 pages/min), scraped 0 items (at 0 items/min)
-    2016-09-01 16:51:28 [scrapy] DEBUG: Crawled (404) <GET http://quotes.toscrape.com/robots.txt> (referer: None)
-    2016-09-01 16:51:28 [scrapy] DEBUG: Crawled (200) <GET http://quotes.toscrape.com/page/1/> (referer: None)
-    2016-09-01 16:51:29 [scrapy] DEBUG: Crawled (200) <GET http://quotes.toscrape.com/page/2/> (referer: None)
-    2016-09-01 16:51:29 [scrapy] INFO: Closing spider (finished)
+Now, check the files in the current directory. You should notice that two new
+files have been created: *quotes-1.html* and *quotes-2.html*, with the content
+for the respective URLs, as our ``parse`` method instructs.
 
-.. note::
-    At the end you can see a log line for each URL defined in ``start_urls``.
-    Because these URLs are the starting ones, they have no referrers, which is
-    shown at the end of the log line, where it says ``(referer: None)``.
+.. note:: If you are wondering why we haven't parsed the HTML yet, hold
+  on, we will cover that soon.
 
-Now, check the files in the current directory. You should notice two new files
-have been created: *quotes-1.html* and *quotes-2.html*, with the content for the respective
-URLs, as our ``parse`` method instructs.
 
 What just happened under the hood?
 ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
 
-Scrapy creates :class:`scrapy.Request <scrapy.http.Request>` objects
-for each URL in the ``start_urls`` attribute of the Spider, and assigns
-them the ``parse`` method of the spider as their callback function.
+Scrapy schedules the :class:`scrapy.Request <scrapy.http.Request>` objects
+returned by the ``start_requests`` method of the Spider. Upon receiving
+a response for each one, it instantiates :class:`scrapy.http.Response`
+objects and calls the ``parse`` callback method passing the response as
```

Comment:
It could read as if Scrapy always calls `parse` callback, not only in our example, because it is written in present simple ('schedules', 'instantiates'), and for `start_requests` it is indeed what always happens.